src/model/window-guardian.ts

```diff
@@ -66,18 +66,67 @@ const makeWindowGuardianConfig = (
     } as WindowGuardianConfig;
 };
 
+export const makeGuardianBrowserCAPI = (CAPI: CAPIType): CAPIBrowserType => ({
+    config: {
+        isDev: process.env.NODE_ENV !== 'production',
+        ajaxUrl: CAPI.config.ajaxUrl,
+        shortUrlId: CAPI.config.shortUrlId,
```

Comment:
So I don't think this needs to happen in this PR but just so we don't forget: things like this (`shortUrlId`) aren't really 'config', they're properties of the article. Now that we have control of this type, it'd be great to have another PR later to review this structure and rationalise things